packages/elemental/elemental/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,51 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,32 +1,40 @@
+ image:
+   empty: rancher/pause:3.1
+-  repository: "elemental-operator"
+-  tag: v1.3.4
++  repository: "rancher/mirrored-elemental-operator"
++  tag: "1.3.4"
+   imagePullPolicy: IfNotPresent
++
+ seedImage:
+-  repository: "elemental-seedimage"
+-  tag: v1.3.4
++  repository: "rancher/mirrored-elemental-seedimage-builder"
++  tag: "1.3.4"
+   imagePullPolicy: IfNotPresent
+-#channel:
+-#  repository: rancher/elemental-teal-channel
+-#  tag: latest
++
++channel:
++  repository: ""
```

Comment:
Why is this empty ?

Reply:
The channel image provides a json struct with URLs and metadata for Elemental-compatible base OS images.
The image we ship has a list of OS entries with a fixed and full IBS URL (we cannot ship SUSE based OS but from suse.com domain).
Moreover, the URL provided in the json may even be an https one: so this means it will break in Rancher airgap environments.
So, the idea is that it is up to the user to provide a valid channel (or live without it and provide OS URLs during elemental usage).
Note however that in the question section, if the user picks up the channel option, we provide a pre-filled default channel pointing to the suse registry. The user may keep it if not in airgap or change it with their own created channel there.